pkg/component/extensions/crds/assets/crd-extensions.gardener.cloud_workers.yaml

```diff
@@ -511,10 +516,16 @@ spec:
                     name:
                       description: Name is the name of the `MachineDeployment` resource.
                       type: string
+                    priority:
+                      description: Priority (or weight) is the importance by which
+                        this machine deployment will be scaled by cluster autoscaling.
+                      format: int32
+                      type: integer
                   required:
                   - maximum
                   - minimum
                   - name
+                  - priority
```

Comment:
Sorry that I missed this in the previous reviews. 
`priority` is set to required here which will cause an error during the reconciliation of the `Worker`, if the worker extensions are not deployed with support to manage the `priority` field. Basically, when the `MachineDeployment` status is added to the `Worker`'s status, it will not contain the `priority` field and the update will be rejected.